server/adapters/bigquery.go

```diff
@@ -236,6 +239,14 @@ func (bq *BigQuery) DropTable(table *Table) error {
 	return nil
 }
 
+//Clean deletes all records in tableName table
+func (bq *BigQuery) Clean(tableName string) error {
+	query := fmt.Sprintf(truncateBigQueryTemplate, bq.config.Project, bq.config.Dataset, tableName)
+	bq.queryLogger.LogQuery(query)
+	_, err := bq.client.Query(query).Read(bq.ctx)
```

Comment:
here we should check returned error. If a table doesn't exist BigQuery returns a specific error.
Jitsu should omit such error and return nil.